// @tsparticles/preset-hyperspace/umd/options.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "@tsparticles/engine"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    exports.options = void 0;
    const engine_1 = require("@tsparticles/engine");
    exports.options = {
        background: {
            color: "#000000",
        },
        particles: {
            number: {
                value: 100,
            },
            color: {
                value: "#ffffff",
            },
            life: {
                count: 1,
                duration: {
                    value: 5,
                },
            },
            shape: {
                type: "circle",
            },
            opacity: {
                value: 1,
            },
            size: {
                value: 3,
            },
            move: {
                enable: true,
                speed: 10,
                decay: 0.005,
                direction: engine_1.MoveDirection.outside,
                straight: true,
                outModes: engine_1.OutMode.destroy,
                trail: {
                    enable: true,
                    length: 15,
                    fill: {
                        color: "#000000",
                    },
                },
            },
        },
        emitters: {
            position: {
                x: 50,
                y: 50,
            },
            size: {
                width: 100,
                height: 100,
            },
            rate: {
                quantity: 10,
                delay: 0.1,
            },
        },
    };
});
