// @tsparticles/preset-fountain/umd/options.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "@tsparticles/engine"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    exports.options = void 0;
    const engine_1 = require("@tsparticles/engine");
    exports.options = {
        fpsLimit: 120,
        particles: {
            bounce: {
                vertical: {
                    value: {
                        min: 0.75,
                        max: 0.85,
                    },
                },
            },
            color: {
                value: ["#3998D0", "#2EB6AF", "#A9BD33", "#FEC73B", "#F89930", "#F45623", "#D62E32", "#EB586E", "#9952CF"],
            },
            number: {
                value: 0,
            },
            destroy: {
                mode: "split",
                split: {
                    count: 2,
                    factor: {
                        value: {
                            min: 1.1,
                            max: 2,
                        },
                    },
                    rate: {
                        value: {
                            min: 2,
                            max: 3,
                        },
                    },
                },
            },
            opacity: {
                value: 0.5,
            },
            size: {
                value: {
                    min: 10,
                    max: 20,
                },
            },
            move: {
                enable: true,
                gravity: {
                    enable: true,
                    maxSpeed: 50,
                },
                speed: {
                    min: 10,
                    max: 20,
                },
                direction: engine_1.MoveDirection.none,
                random: false,
                straight: false,
                outModes: {
                    bottom: engine_1.OutMode.split,
                    default: engine_1.OutMode.bounce,
                    top: engine_1.OutMode.none,
                },
                trail: {
                    enable: true,
                    fill: {
                        color: "#fff",
                    },
                    length: 3,
                },
            },
        },
        detectRetina: true,
        background: {
            color: "#fff",
        },
        emitters: {
            direction: engine_1.MoveDirection.top,
            life: {
                count: 0,
                duration: 0.15,
                delay: 3,
            },
            rate: {
                delay: 0.1,
                quantity: 5,
            },
            size: {
                width: 0,
                height: 0,
            },
        },
    };
});
